src/coreclr/jit/codegenarm64.cpp

```diff
@@ -1817,6 +1817,26 @@ void CodeGen::genCodeForMulHi(GenTreeOp* treeNode)
     genProduceReg(treeNode);
 }
 
+// Generate code to get the 64-bits of a 32*32 bit multiplication result
+void CodeGen::genCodeForMulWide(GenTreeOp* treeNode)
```

Comment:
Isn't this a `GT_MUL_LONG`? We only use for 32 bit targets now, but it should be straightforward to use it on ARM64 too.